### DISCUSSION TITLE
Limit AWS CIS scan to Level 1 only?

### DISCUSSION THREAD

**Discussion by itijeffhoover** [prowler-cloud/prowler#3017] [Q&A] [answered]
Hello,

We finally migrated from v2 to v3 and we are wondering if there is a way in v3 to limit the CIS Benchmark scans to Level 1 only?  We don't need to meet Level 2, so those checks use extra compute capacity to scan and generate unnecessary API calls and entries in the scan results.  Does that exist in v3 like it did in v2?

Thank you

**Comment by MrCloudSec** [comment, marked as answer]
Hi @itijeffhoover , good idea! I have include this feature in our roadmap. Thanks for reaching us out!

**Reply by itijeffhoover** [reply]
Thank you for adding it to the roadmap.
